spec.html

```diff
@@ -27137,31 +27143,257 @@
+            </table>
+          </emu-table>
+          <emu-table id="sec-todatestring-month-names" caption="Names of months of the year">
+            <table>
+              <tbody>
+              <tr>
+                <th>
+                  Number
+                </th>
+                <th>
+                  Name
+                </th>
+              </tr>
+              <tr>
+                <td>
+                  0
+                </td>
+                <td>
+                  `"Jan"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  1
+                </td>
+                <td>
+                  `"Feb"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  2
+                </td>
+                <td>
+                  `"Mar"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  3
+                </td>
+                <td>
+                  `"Apr"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  4
+                </td>
+                <td>
+                  `"May"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  5
+                </td>
+                <td>
+                  `"Jun"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  6
+                </td>
+                <td>
+                  `"Jul"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  7
+                </td>
+                <td>
+                  `"Aug"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  8
+                </td>
+                <td>
+                  `"Sep"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  9
+                </td>
+                <td>
+                  `"Oct"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  10
+                </td>
+                <td>
+                  `"Nov"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  11
+                </td>
+                <td>
+                  `"Dec"`
+                </td>
+              </tr>
+              </tbody>
+            </table>
+          </emu-table>
+        </emu-clause>
+
+        <emu-clause id="sec-timezoneestring" aoid="TimeZoneString">
+          <h1>Runtime Semantics: TimeZoneString( _tv_, )</h1>
+          <p>The following steps are performed:</p>
+          <emu-alg>
+            1. Assert: Type(_tv_) is Number.
+            1. Assert: _tv_ is not *NaN*.
+            1. Let _offset_ be (LocalTZA + DaylightSavingTA(_tv_)) / msPerSecond.
```

Comment:
You're right; this is written against the current spec, but needs to be updated against the other one (based on whichever lands second). I guess rather than LocalTZA, we'd use LocalTimeZoneAdjustment(_tz_, true), right?